examples/dreamcast/basic/threading/spinlock_test/spinlock_test.c

```diff
@@ -0,0 +1,153 @@
+/* KallistiOS ##version##
+
+   spinlock_test.c
+
+   (c) 2023 Lawrence Sebald
+   (c) 2023 Falco Girgis
```

Comment:
Not sure why I'm being credited here...

```suggestion
   Copyright (C) 2023 Falco Girgis
```